deltachat-ffi/deltachat.h

```diff
@@ -7569,9 +7569,14 @@ void dc_event_unref(dc_event_t* event);
 
 /// "Could not yet establish guaranteed end-to-end encryption, but you may already send a message."
 ///
-/// Used as info message.
+/// @deprecated 2025-03
 #define DC_STR_SECUREJOIN_WAIT_TIMEOUT 191
 
+/// "This takes longer than expected, maybe devices are offline…\n\nHowever, the process continues in background, you can do something else"
```

Comment:
Maybe "the peer is offline"? Otherwise it's not clear whose devices are meant and how many of them are offline

Reply:
What about "the device is offline" or "one of the devices is offline"? The plural also surprised me when I first read this message. I think it's clear enough which device is meant.

(but yes, the discussion here doesn't need to block merging this PR here)